include/boost/histogram/algorithm/empty.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 Hans Dembinski and Henry Schreiner
+//
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt
+// or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_HISTOGRAM_ALGORITHM_EMPTY_HPP
+#define BOOST_HISTOGRAM_ALGORITHM_EMPTY_HPP
+
+#include <boost/histogram/coverage.hpp>
```

Comment:
There is no such header, that's why the tests fail.

Reply:
Oops, missed that. I moved it to a new header, then realized it wasn't required for this PR since I'm using `indexed`, so left that off the PR. Forget I imported it here. My computer still has the header so it doesn't fail locally.

Reply:
Since the coverage enum is used by several algorithms now, it should indeed be moved into another header, but the header should be boost/histogram/algorithm/coverage.hpp

Reply:
Actually, no, it should just live in fwd.hpp.

Reply:
It seems odd to have something live in fwd that doesn't live anywhere else...

Reply:
Though, since it is always part of a function call, and can't be used on its own, I can see why it could make sense there.